src/main/features/core/autoUpdater.js

```diff
@@ -57,7 +64,7 @@ const checkUpdateServer = () => {
   https.get('https://update.gpmdp.xyz', () => {
     setUpAutoUpdate();
   }).on('error', () => {
-    console.log('################### !! Update server down !! ##################'); // eslint-disable-line
+    Logger.error('################### !! Update server down !! ##################');
```

Comment:
I would want this to go to the console aswell, not sure if that is how winston works or not